src/server.h

```diff
@@ -1017,7 +1017,7 @@ struct redisMemOverhead {
     size_t lua_caches;
     size_t overhead_total;
     size_t dataset;
-    size_t total_keys;
+    long long total_keys;
```

Comment:
unsigned long long?

Reply:
I originally wanted to use unsigned long long, but I looked at similar code, so I used long long instead.
like emptyDbStructure method,

Reply:
I'm not really sure any of those variable types were picked for a reason. Total keys can't be negative, I'm not really sure this shouldn't be unsigned long either.

Reply:
```c
long long emptyDb(int dbnum, int flags, void(callback)(void*)) {
    int async = (flags & EMPTYDB_ASYNC);
    RedisModuleFlushInfoV1 fi = {REDISMODULE_FLUSHINFO_VERSION,!async,dbnum};
    long long removed = 0;

    if (dbnum < -1 || dbnum >= server.dbnum) {
        errno = EINVAL;
        return -1;
    }

```
I want to change return type of emptyDb to unsigned long long, but there is a problem.
The emptyDb method returns -1 when an error dbid is passed in, and the return value of the emptyDb unused anywhere, but I'm not sure if there are any other side effects.